source/features/batch-mark-files-as-viewed.tsx

```diff
@@ -18,23 +19,20 @@ function isChecked(file: HTMLElement): boolean {
 }
 
 function batchToggle(event: delegate.Event<MouseEvent, HTMLFormElement>): void {
-	if (!event.shiftKey || !previousFile) {
+	if (!event.shiftKey) {
 		return;
 	}
 
 	event.preventDefault();
 	event.stopImmediatePropagation();
 
-	const previousFileState = isChecked(previousFile);
-	const thisFile = event.delegateTarget.closest('.js-file')!;
 	const files = select.all('.js-file');
-	const selectedFiles = files.slice(...[
-		files.indexOf(previousFile) + 1,
-		files.indexOf(thisFile) + 1,
-	].sort((a, b) => a - b));
+	const thisFile = event.delegateTarget.closest('.js-file')!;
+	const isThisFileChecked = isChecked(thisFile);
 
+	const selectedFiles = getItemsBetween(files, previousFile ?? files[0], thisFile);
```

Comment:
If you're going to always provide the previous file, then we don't need to duplicate the logic in `getItemsBetween` with `| undefined` and the ternary operator